[Feature] sc-25552 Comment test report to PR

kentwelcome opened the pull request:
- upload test result to artifact

Signed-off-by: Kent Huang <kentwelcome@gmail.com>

kentwelcome committed:
[Feature] sc-25552 Comment test report to PR

 - upload test result to artifact

Signed-off-by: Kent Huang <kentwelcome@gmail.com>

neighborhood999 commented on index.js:
```diff
@@ -0,0 +1,95 @@
+
+  // Upload artifacts
+  const artifactClient = artifact.create();
+  const artifactName = 'piperide-cli-test-report';
```
A typo

```suggestion
  const artifactName = 'piperider-cli-test-report';
```

kentwelcome replied:
Fix! Thx for reivewing

kentwelcome committed:
[Feature] sc-25652 Add GITHUB_ACTION_URL into entrypoint.sh

Signed-off-by: Kent Huang <kentwelcome@gmail.com>

kentwelcome committed:
[Fix] Fix code review typo issue

Signed-off-by: Kent Huang <kentwelcome@gmail.com>

kentwelcome committed:
[Feature] sc-25552 upgrade to use pipeline-cli v0.1.1

 - Add GITHUB_ACTION_URL as external_url in metadata
 - Add --one-json
 - Set the Piperider API server URL

Signed-off-by: Kent Huang <kentwelcome@gmail.com>
